easyaudit/admin_helpers.py

```diff
@@ -51,6 +58,11 @@ def user_link(self, obj):
     def has_add_permission(self, request, obj=None):
         return False
 
+    def has_delete_permission(self, request, obj=None):
+        if settings.READONLY_EVENTS:
```

Comment:
Minor (non-blocking):
```suggestion
        return not settings.READONLY_EVENTS
```